amxmodx/CLang.cpp

```diff
@@ -331,15 +332,10 @@ int CLangMngr::MergeDefinitionFile(const char *file)
 				break;
 			}	
 		}
-		AMXXLOG_Log("[AMXX] Failed to open dictionary file: %s", file);
+		AMXXLOG_Log("[AMXX] There was a problem opening or hashing dictionary file: %s", file);
```

Comment:
Keep the original message error. At this point, it can be only that since file is unlikely to be empty and you pass manually the constant.